specs/test_2.js

```diff
@@ -0,0 +1,30 @@
+let MainPage = require("../pages/main.page");
+let LoginPage = require("../pages/login.page");
+let RegistrationPage = require("../pages/registration.page");
+
+describe('Protractor Demo App', () => {
+    it('Registration test', async () => {
```

Comment:
Please review each test and update it name(`it('Registration test', async () => {`) to fit the test purpose.